2020-APR/answer.md

```diff
@@ -0,0 +1,203 @@
+<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
+
+# AACS1084 April 2020 Answers
+
+[Link to the paper](https://eprints.tarc.edu.my/14889/1/AACS1084.pdf)
+
+- [Question 1](#question-1)
+- [Question 2](#question-2)
+- [Question 3](#question-3)
+- [Question 4](#question-4)
+
+## Answers
+
+### Question 1
+
+a) (i)
+
+Structure is a data structure that allows user to store a group of variables with different data types.
+
+b)
+
+To group several related variables taht is different in data types into one place.
+
+c) (i)
+
+```c
+typedef struct {
+  char treatmentName[30];
+  char footSoak;
+  double price;
+} Treatment;
+```
+
+c) (ii)
+
+```c
+Treatment treatmentM = {"Original Urutan Malaysia", 'Y', 268.0};
+```
+
+c) (iii)
+
+```c
+typedef struct {
+  int code;
+  char name[20];
+  Treatment treat;
+} Customer;
+```
+
+c) (iv)
+
+```c
+Customer custA = {1277, "Mandy", "Herbal Massage", 'N', 188.0};
+```
+
+(c) (v)
+
+```c
+if (custA.code == 1277)
+  custA.treat.price = 150.4;
+```
+
+### Question 2
+
+a)
+
+Text files store human readable characters.
+Each line in the text file ends with newline(\n) character.
+Text files are easily readable and modifiable.
```

Comment:
use point form, else it render as single paragraph, weird to read

last point make it like this: Text files are easily readable and modifiable using text editor.